classes/book.rb

```diff
@@ -0,0 +1,15 @@
+require './item'
+
+class Book < item
+  def initialize(publisher, cover_state, publish_date)
+    super(publish_date)
```

Comment:
I noticed an error here because the provided argument is less than what the parent class expects. Please fix this 👍